public/search.js

```diff
@@ -41,9 +47,9 @@ function appendData(data) {
 
 function nextHandler() {
     page_no++;
-    document.getElementById('next').href = `/search?q=${input}&page=${page_no}`
+    document.getElementsByClassName('next').href = `/search?q=${input}&page=${page_no}`
     page_no = page_no - 2;
-    document.getElementById('previous').href = `/search?q=${input}&page=${page_no}`
+    document.getElementsByClassName('previous').href = `/search?q=${input}&page=${page_no}`
```

Comment:
little idea here, but within CSS in general, indeed using class makes more sense than id!